bundles/core/src/test/resources/title/exporter-title-jcr-title-type.json

```diff
@@ -10,5 +10,8 @@
             "dc:title": "Hello World"
         }
     },
+    "link": {
```

Comment:
Invalid links should return `null` so that they don't get rendered in JSON export.